src/ui/components/Rating/index.js

```diff
@@ -82,18 +101,18 @@ export class RatingBase extends React.Component {
   }
 
   render() {
-    const { className, i18n, rating, readOnly, styleSize, isOwner } = this.props;
+    const { className, i18n, isOwner, rating, readOnly, styleSize } = this.props;
     if (!RATING_STYLE_SIZES.includes(styleSize)) {
       throw new Error(
         `styleSize=${styleSize} is not a valid value; ` +
         `possible values: ${RATING_STYLE_SIZES.join(', ')}`);
     }
-    let description = i18n.gettext('This add-on has no ratings.');
-    if (rating) {
-      description = i18n.sprintf(i18n.gettext('Rated %(rating)s out of 5'),
-        { rating: i18n.formatNumber(parseFloat(rating).toFixed(1)) });
-    } else if (!readOnly) {
-      description = i18n.gettext('Click to rate this add-on');
+
+    let description = null;
+    // Wrap readOnly ratings with a description to maintain functionality
+    // for the "Average rating of developer’s add-ons" tooltip
+    if (readOnly) {
```

Comment:
This could be shortened to `const description = readOnly ? this.renderTitle(i18n, rating, readOnly, null) : null;`